lib/ecto/adapters/sql.ex

```diff
@@ -303,7 +317,7 @@ defmodule Ecto.Adapters.SQL do
   @spec rollback_test_transaction(Ecto.Repo.t, Keyword.t) :: :ok
   def rollback_test_transaction(repo, opts \\ []) do
     pool = pool!(repo)
```

Comment:
very good idea actually, I will adapt this and related tests. Sorry for not noticing this before.